Source/IntegrationTests/TestFiles/LitTests/LitTest/exports/ExportResolve.dfy.expect

```diff
@@ -94,8 +94,7 @@ ExportResolve.dfy(288,39): Error: member 'u' has not been imported in this scope
 ExportResolve.dfy(290,18): Error: type of corresponding source/RHS (G.Trait) does not match type of bound variable (object?)
 ExportResolve.dfy(290,46): Error: type of corresponding source/RHS (G.Klass) does not match type of bound variable (object?)
 ExportResolve.dfy(298,19): Error: member '_ctor' has not been imported in this scope and cannot be accessed here
-ExportResolve.dfy(342,15): Error: unresolved identifier: reveal_OpaqueFunction
-ExportResolve.dfy(342,29): Error: expression has no reveal lemma
+ExportResolve.dfy(342,15): Error: cannot reveal 'OpaqueFunction' because no revealable constant, function, assert label, or requires label in the current scope is named 'OpaqueFunction'
```

Comment:
Not blocking, but I thought it should not be an error but a warning. Otherwise, it makes experimenting with opaque to be hard.